code/dynamic_programming/largest_sum_contiguous_subarray/LargestSum.java

```diff
@@ -0,0 +1,15 @@
+import java.util.*;
+
+public class LargestSum{
+    public static void main(String[] args) {
+        int input[] = new int[5];
+        int sum = 0;
+        for (int i=0 ; i<5; i++){
+            Scanner scanner = new Scanner(System.in);
+            input[i] = scanner.nextInt();
+            sum = sum + input[i];
+        }
+        Arrays.sort(input);
```

Comment:
This does not do what is intended. It just adds up all numbers except the smallest one

See [this](https://github.com/OpenGenus/cosmos/blob/master/code/dynamic_programming/largest_sum_contiguous_subarray/Largest_sum_continuous_array.java) for guidance 👍